signac/contrib/project.py

```diff
@@ -142,7 +142,12 @@ def __init__(self, config=None):
         self._config = _ProjectConfig(config)
 
         # Ensure that the project id is configured.
-        self.get_id()
+        id_ = self.get_id()
+        if id_ is None:
+            raise LookupError(
+                "Unable to determine project id ."
+                "Are you sure '{}' is a signac project path?".format(
```

Comment:
I’m generally not a fan of questions in error messages. I would prefer an instruction like “Please verify that... is a signac project path.” I know we use the question style elsewhere, this is just personal preference.